### PR TITLE
css-canvas-medical | Joseph Nguyen

### PR DESCRIPTION
**Pull request by josephtnguyen**
canvas

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Joseph! Please attach screenshots showing your entire page, I can only see the top half

**Comment by josephtnguyen** [comment]
canvas-2

File: css-canvas-medical/styles.css [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+p {
+  font-size: 13px;
+  line-height: 1.4rem;
+  margin: 0;
+}
+
+.hero-heading,
+.hero-par {
+  padding-right: 46%;
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:104]: That's a lot of padding!! Why is this necessary? There is likely a better way to accomplish this.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work on this!!! One note for you:
